library/tasks/MergeSortBam.wdl

```diff
@@ -1,18 +1,17 @@
 
 task MergeSortBamFiles {
-  Array[Array[File]] bam_inputs
+  Array[File] bam_inputs
+  String sort_order
+
   Int disk_size = 500
   Int compression_level = 5
 
   command {
-    input_line=$(python -c "print(' INPUT='.join('${sep=' ' bam_inputs}'.replace('[', '').replace(']', '').replace(',', '').split()))")
-
-    echo $input_line
     java -Dsamjdk.compression_level=${compression_level} -Xms7000m -jar /usr/gitc/picard.jar \
       MergeSamFiles \
       USE_THREADING=true \
-      SORT_ORDER=coordinate \
-      INPUT=$input_line \
+      SORT_ORDER=${sort_order} \
+      INPUT=${sep=' INPUT=' bam_inputs} \
```

Comment:
Won't this insert ` INPUT=` in between each item in the `bam_inputs` array instead of joining the items in the array with ` INPUT=` as a prefix like it was previously?

Reply:
I believe the python was doing the same thing as this `sep` command.  Splitting the `Array[Array[File]] bam_inputs` into a string with spaces between each element by `.replace()`ing all `[`, `]`, `,` characters with empty characters.  It then `.split()`s that string by spaces to get a python array and then `.join()`s that array with " INPUT=".  Because I was able to change the `Array[Array[File]] bam_inputs` input to `Array[File] bam_inputs` it was simpler to use wdls `sep` to accomplish the same thing.

Reply:
Oh I see now! I misunderstood what it was doing before. Thanks for clarifying!